infra/modules/ecs/tasks.tf

```diff
@@ -45,3 +45,33 @@ resource "aws_ecs_task_definition" "hello-world" {
     }
   ])
 }
+
+resource "aws_security_group" "hello-world-task-sg" {
+  name                   = "${var.project_name}-hello-world-task-sg-${var.environment}"
+  description            = "Security group for hello world ecs service"
```

Comment:
Replace service with task?

Is this the same resource in terraform or did you bin off the old one and create this one?

Reply:
Good catch thanks - probably from before I understood the difference between task and service :)

Yeah I just moved this resource from the services.tf file. It was used by the service, and is now used by the lambda; this file made the most sense as a home for it.